XamlControlsGallery/ControlPages/RichEditBoxPage.xaml

```diff
@@ -140,11 +140,6 @@
                                         <Rectangle Fill="Gray"/>
                                     </Button.Content>
                                 </Button>
-                                <Button Click="ColorButton_Click" AutomationProperties.Name="Black">
```

Comment:
Why remove this color only?

Reply:
This is from integrating #524 that was originally made for the WinUI 2 version.